portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileShortcutLocalServiceImpl.java

```diff
@@ -505,6 +510,21 @@ protected void validate(User user, long toFileEntryId)
 		}
 	}
 
+	protected void validateFolder(long groupId, long folderId)
+		throws PortalException {
+
+		if (folderId != DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
+			DLFolder parentDLFolder = _dlFolderPersistence.findByPrimaryKey(
```

Comment:
Can we use localService or service instead of persistence here? Normally we use persistence for the entity of the service, in this case DLFileShortcut.

Reply:
I don't mind changing it but I was copying it from DLFileEntryLocalServiceImpl - https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java#L4049-L4050

Let me know.

Reply:
If we already have it like this, perfect then 👍. Thanks!